src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -79,6 +80,7 @@ private Properties buildProperties() {
         props.setProperty("resource.loader", "class");
         props.setProperty("class.resource.loader.class", ClasspathResourceLoader.class.getCanonicalName());
         props.setProperty("runtime.log", new File(configuration.getReportDirectory(), "velocity.log").getPath());
+        props.setProperty("eventhandler.referenceinsertion.class", "org.apache.velocity.app.event.implement.EscapeHtmlReference");
```

Comment:
Mmh. I don't understand why you call it 'unchanged'

Suppose I have a json file that contains:

    <b> a bold statement </b>

When rendered by a plain text reporter, like the command-line, it will be rendered as 

    <b> a bold statement </b>

And when rendered by a html reporter that doesn't escape the input
<b> a bold statement </b>
Because now I've lost the \<b\> \</b\> tags from the report. 

---

I also don't understand why you expect the html to be escaped in the json file if it should be displayed escaped. When creating the json file and escaping html I've coupled that json file to a particular report formatter (one that uses html). This seems bad practice.